pages/instructorAssessments/instructorAssessments.js

```diff
@@ -38,6 +38,7 @@ router.get('/', function(req, res, next) {
         course_instance_id: res.locals.course_instance.id,
         authz_data: res.locals.authz_data,
         req_date: res.locals.req_date,
+        assessments_group_by: res.locals.course.assessments_group_by,
```

Comment:
I personally would prefer to have this coming from `pl_courses` itself in the query, though I wouldn't mind being convinced otherwise.

Reply:
This value is already being authorized by the middleware, and it saves a few lines of SQL joins, I think they're both just as valid